core/dbt/clients/jinja.py

```diff
@@ -231,9 +231,10 @@ def get_macro(self):
         template = self.get_template()
         # make the module. previously we set both vars and local, but that's
         # redundant: They both end up in the same place
+        # make_module is in jinja2.environment. It returns a TemplateModule
         module = template.make_module(vars=self.context, shared=False)
-        macro = module.__dict__[get_dbt_macro_name(name)]
         module.__dict__.update(self.context)
+        macro = module.__dict__[get_dbt_macro_name(name)]
```

Comment:
is the switching of this line and the one above it significant in some way?

Reply:
I think it was an accident since it's a no-op. Can put back to avoid the noise.

Reply:
Actually it might not be a no-op, but I no longer remember if there was a reason. Will test it.